main.py

```diff
@@ -46,39 +47,148 @@
 import argparse
 import re
 import tuf
+import hashlib
+import json
 
 TOTO_TOOL_VERSION = "Toto Build/Test Metadata Generator 0.4"
 DEFAULT_POLICY_FILENAME = "default_policy.json"
+DEFAULT_CUR_DIR = os.getcwd()
+DEFAULT_WORK_DIR = os.path.join(DEFAULT_CUR_DIR, "work")
 
 def main():
+  # Setup the command line parser
+  args = get_command_line_args()
```

Comment:
In order to support commands passed via the command policy file, the setup of the arg-parser has to be altered to make the COMMAND argument optional. Right now this arg is mandatory and it doesn't make sense to include it if commands are being read from the policy file.